crates/pop-cli/src/main.rs

```diff
@@ -36,40 +36,44 @@ enum Commands {
 	#[clap(alias = "t")]
 	#[cfg(feature = "contract")]
 	Test(commands::test::TestArgs),
+	/// Install prerequisites for Substrate development
```

Comment:
I like _Set up the environment for Substrate development_ more. But up to you to decide